src/test/simple/lender.sol

```diff
@@ -25,10 +25,14 @@ contract SimpleLender {
     LenderTokenLike public tkn;
     LenderTokenLike public collateral;
 
+
+    address public backer;
```

Comment:
In the simple adapter, the backer is the contract itself.

Reply:
Why does the SimpleLender even need a backer?

Reply:
We are only using Simple lender for testing and I wanted to run the system test on both.

I wanted a way to check if the adapter received currency and collateral. In the simpleLender case,  the backer is the contract itself.

Reply:
I can remove it. I could just overwrite the check method too. (to support two different way)

Reply:
fixed